app/containers/App/reducer.js

```diff
@@ -10,6 +10,7 @@ import {
 const initialState = fromJS({
   modalStack: [],
   modalCloseHandlers: [],
+  modalBackdrop: [],
```

Comment:
What if we want to add some new option for modals? Let's add the key `modalOptions` and put `closeHandler` and `backdrop` options into it. What do you think?

Reply:
agree with you.
done.